Tests/SwiftStdlibTests/StringExtensionsTests.swift

```diff
@@ -565,26 +565,44 @@ final class StringExtensionsTests: XCTestCase {
         XCTAssertEqual(url, "it's%20easy%20to%20encode%20strings")
     }
 
-    func testMatches() {
+    let emailPattern = "[A-Z0-9a-z._%+-]+@[A-Za-z0-9.-]+\\.[A-Za-z]{2,}"
+
+    func testPatternMatches() {
         XCTAssert("123".matches(pattern: "\\d{3}"))
         XCTAssertFalse("dasda".matches(pattern: "\\d{3}"))
-        XCTAssertFalse("notanemail.com".matches(pattern: "[A-Z0-9a-z._%+-]+@[A-Za-z0-9.-]+\\.[A-Za-z]{2,}"))
-        XCTAssert("email@mail.com".matches(pattern: "[A-Z0-9a-z._%+-]+@[A-Za-z0-9.-]+\\.[A-Za-z]{2,}"))
+        XCTAssertFalse("notanemail.com".matches(pattern: emailPattern))
+        XCTAssert("email@mail.com".matches(pattern: emailPattern))
+    }
+
+    func testRegexMatches() {
+        // swiftlint:disable force_try
+        XCTAssert("123".matches(regex: try! NSRegularExpression(pattern: "\\d{3}")))
```

Comment:
I know it was like this before, but I think will be an improvement to use `XCTAssertTrue` :)